mp/src/game/shared/hl2mp/hl2mp_playeranimstate.cpp

```diff
@@ -413,13 +413,15 @@ bool CHL2MPPlayerAnimState::HandleSwimming( Activity &idealActivity )
 bool CHL2MPPlayerAnimState::HandleMoving( Activity &idealActivity )
 {
 #ifdef NEO
-	if (m_pHL2MPPlayer->IsWalking())
+	Vector vecVelocity;
+	GetOuterAbsVelocity(vecVelocity);
+	if (m_pHL2MPPlayer->IsSprinting())
```

Comment:
We could probably move the `vecVelocity` math inside the `else` condition scope, so it doesn't need to be calculated for the `->IsSprinting()` case.